atomix/cluster/src/test/java/io/atomix/raft/RaftRule.java

```diff
@@ -565,11 +566,9 @@ private void appendEntryAsync(final RaftServer leader, final int entrySize) {
   private TestAppendListener appendEntry(final int entrySize, final LeaderRole leaderRole) {
     final var appendListener = new TestAppendListener();
     position += 1;
-    leaderRole.appendEntry(
-        position,
-        position + 10,
-        ByteBuffer.wrap(RandomStringUtils.random(entrySize).getBytes()),
-        appendListener);
+    final var bytes = new byte[entrySize];
+    random.nextBytes(bytes);
+    leaderRole.appendEntry(position, position + 10, ByteBuffer.wrap(bytes), appendListener);
```

Comment:
:thought_balloon: I'm wondering a bit about this magic number `10` :sweat_smile:

Reply:
Same :smile: I'm guessing that this is not necessary for most(all?) tests.